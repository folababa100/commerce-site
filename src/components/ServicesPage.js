import React from 'react';
import Header from './Header';

const ServicesPage = (props) => {
  return (
    <div>
      <Header/>
    </div>
  )
}

export default ServicesPage;
