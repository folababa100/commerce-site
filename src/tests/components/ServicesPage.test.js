import React from 'react';
import { shallow } from "enzyme";
import ServicesPage from '../../components/ServicesPage';

test('should correctly render component', () => {
  const wrapper = shallow(<ServicesPage/>);
  expect(wrapper).toMatchSnapshot()
})

