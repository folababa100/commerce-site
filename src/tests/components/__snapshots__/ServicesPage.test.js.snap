// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render component 1`] = `
<div>
  <Connect(Header) />
  <ServiceDesign />
</div>
`;
